.github/workflows/lint-and-test.yml

```diff
@@ -57,4 +57,33 @@ jobs:
 
     - name: test
       run: yarn test
-      
\ No newline at end of file
+      
+  init-e2e-test-check:
+    runs-on: ubuntu-latest
+
+    steps: 
+      - uses: actions/checkout@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: |
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
+            **/node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+    
+      - name: dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install
+
+      - name: creating e2e tests check-run
+        env:
+          GITHUB_CONTEXT_EVENT: ${{ toJson(github.event) }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: yarn e2e:create-check-run
```

Comment:
```suggestion
        run: yarn e2e:create-check-run

```
😁

Reply:
I always get stuck with EOF and YML files (VS Code doesn't do it for me) 😅